src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const myList = document.querySelector('.tree');
+
+const listItems = myList.getElementsByTagName('li');
+
+for (let i = 0; i < listItems.length; i++) {
```

Comment:
Consider using a `forEach` loop instead of a traditional `for` loop for better readability and ease of use.